src/components/MoveButton.vue

```diff
@@ -1,13 +1,18 @@
 <template>
-  <button class='btn btn-secondary mr-2 mb-2 move-button' @click="handleClick" @mouseenter="hover" @mouseleave="leave" :title="button.tooltip" v-b-tooltip.html>
-    <slot></slot>
-  </button>
+  <div>
+    <button class='btn btn-secondary mr-2 mb-2 move-button' @click="handleClick" @mouseenter="hover" @mouseleave="leave" :title="button.tooltip" >
+      <slot></slot>
+    </button>
+    <b-modal v-model="modalShow" @ok="handleOK" :title="button.label" ok-title="OK, I pick this one!">
```

Comment:
should probably add `size='lg'`, for Ivits

Reply:
Every faction deserves `lg` ;)